classes/submission/DAO.php

```diff
@@ -79,11 +81,12 @@ public function getExportable(int $contextId, $pubIdType = null, $title = null,
                         ->where('pss.setting_name', '=', $pubIdSettingName)
                 )
             )
-            ->where('s.status', '=', Submission::STATUS_PUBLISHED)
             ->where('s.context_id', '=', $contextId)
+            ->where('p.version_stage', '=', VersionStage::VERSION_OF_RECORD)
+            ->where('p.status', '=', Publication::STATUS_PUBLISHED)
```

Comment:
@kaitlinnewson, for you a note, that I also made this change -- to consider only published VoR for Article export as well. Also, I do not check the submission status any more, only publication status. I think they already can diverge, or it will surely be so in the near future.
So maybe just for you to consider this when testing Zenodo.
Thanks!